src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -49,6 +49,7 @@ async function boot(userOptions?: Partial<CircuitStartOptions>): Promise<void> {
     const reconnection = existingConnection || await initializeConnection(options, logger, circuit);
     if (!(await circuit.reconnect(reconnection))) {
       logger.log(LogLevel.Information, 'Reconnection attempt to the circuit was rejected by the server. This may indicate that the associated state is no longer available on the server.');
+      options.reconnectionHandler!.onConnectionRejected(options.reconnectionOptions);
```

Comment:
I suspect we should be checking whether `reconnectionHandler` or `onConnectionRejected` are unset, and skipping the call if so. Or is there some way we know for sure that both of them are set?

Example (untested):

```suggestion
      options.reconnectionHandler?.onConnectionRejected?(options.reconnectionOptions);
```